demodata/getYouTubeUrls.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Code à executer dans une ardoise javascript ici : https://www.youtube.com/results?search_query=snowboard+tricks
```

Comment:
I don't understand the goal of this file nor the `demodata/youtube_videos` file.
If you want to display some content, why don't you create some `Trick` fixtures?

Reply:
I did create a TrickFixtures which get youtube videos urls from this file.
What's wrong? Should I move this file into the DataFixtures folder?